SportsHub/SportsHub.DAL/Migrations/20221019101502_ArticleAndRelatedEntities.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace SportsHub.DAL.Migrations
+{
+    public partial class ArticleAndRelatedEntities : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "ArticleId",
+                table: "Users",
+                type: "int",
+                nullable: true);
```

Comment:
does it a correct migration? I am not sure that we need ArticleId in the User table
Article should keeps an UserId

Reply:
Also, please, don`t do a few migrations for the one commit. It is really hard to review. Drop all added here and then add one new, please.
By the way, better not to have a Pull request with a lot of functionality. It is really hard to review, and there may be missed some important thinks. If you have a lot of issues dependent on each other, just create branches from another branch, not a master!!!

Reply:
Understood